lib/ansible/modules/cloud/amazon/route53_info.py

```diff
@@ -417,7 +417,7 @@ def main():
         ], required=True),
         change_id=dict(),
         hosted_zone_id=dict(),
-        max_items=dict(type='int'),
+        max_items=dict(),
```

Comment:
Please do this for next_marker since that was changes from a str too in the same PR.

Reply:
Ah thx, I missed that.